Packs/GitHub/Integrations/GitHub/GitHub.py

```diff
@@ -1309,6 +1323,25 @@ def get_workflow_usage(owner_name, repository_name, workflow_id):
     return workflow_usage
 
 
+def get_team_members_command():
+    args = demisto.args()
+    org = args.get('organization')
+    team_slug = args.get('team_slug')
+    response = get_team_members(org, team_slug)
+    members = []
+    for member in response:
+        ec_object = {
+            'ID': member.get("id"),
+            'Login': member.get("login"),
+        }
+        members.append(ec_object)
+    ec = {
+        'GitHub.TEAMMEMBER(val.ID === obj.ID)': members
+    }
+    human_readable = tableToMarkdown(f'Team Member of team {team_slug} in organization {org}', t=members, removeNull=True)
+    return_outputs(readable_output=human_readable, outputs=ec, raw_response=response)
```

Comment:
1. Let's switch to `return_results`.
2. Add support for "empty" group, we wouldn't want to send something to the context that is empty.